src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -132,7 +146,7 @@ def create(input_dataset, target, feature=None,
     dataset = _extensions._drawing_classifier_prepare_data(
         input_dataset, feature) if is_stroke_input else input_dataset
 
-    column_names = ['Iteration', 'Loss', 'Elapsed Time']
+    column_names = ['Training Step', 'Loss', 'Validation Accuracy', 'Elapsed Time']
```

Comment:
I think `Iteration` makes more sense than `Training Step`. Displaying `Training Accuracy` would be good too. I'm not sure there is much value in showing `Loss`.

Reply:
I avoided saying "Iteration" because of the `max_iterations` parameter where an iteration is used to refer to an epoch. What do you think? I'll add a Training Accuracy column.

Reply:
Oh, I'm confused. I thought you were printing out one row per iteration/epoch. . . . .  I would recommend doing that rather than print out every ten seconds or so.

Reply:
Done.